code/BuddyTrackerNorth/BuddyTrackerNorth.ino

```diff
@@ -70,11 +70,11 @@ void loop() {
 }
 
 int orientation2LED(float orientation){
-    int fov = 180;
-    int degreesPerLED = fov / STRIP_LENGTH;
+    const int fov = 180;
+    const int degreesPerLED = fov / STRIP_LENGTH;
     int offsetOrientation = orientation + fov / 2;
 
-    if(offsetOrientation > 360){
```

Comment:
whenever unwrapping something, while is better. Just have to make sure stop criteria is okay. This is where the previous calculation may cause problems if something is rounded up that should not be.